ocaml/typing/typedtree.mli

```diff
@@ -228,7 +228,7 @@ and exp_extra =
          *)
   | Texp_poly of core_type option
         (** Used for method bodies. *)
-  | Texp_newtype of string * Jkind.annotation option
+  | Texp_newtype of Ident.t * string loc * Jkind.annotation option * Uid.t
```

Comment:
Please add some comments here (and in `fp_newtypes`) explaining that the new fields are actually unused, but they're necessary for lining up with merlin. Otherwise someone might come along and delete them.